packages/constants/docs/LOG_LEVELS.md

```diff
@@ -0,0 +1,18 @@
+# Log Levels
+
+In order to promote consistant logging across the TV Kitchen ecosystem, the following log levels should be used as described when logging:
+
+- **fatal:** Any error that is forcing a shutdown of the service or application to prevent data loss (or further data loss). Reserve these only for the most heinous errors and situations where there is guaranteed to have been data corruption or loss.
```

Comment:
```suggestion
- **falafel:** Any error that is forcing a shutdown of the service or application to prevent data loss (or further data loss). Reserve these only for the most heinous errors and situations where there is guaranteed to have been data corruption or loss.
```